boost-common/src/main/java/io/openliberty/boost/common/utils/BoostUtil.java

```diff
@@ -88,38 +90,46 @@ private static String getSecurityUtilCmd(String libertyInstallPath) {
         }
     }
 
-    public static String encrypt(String libertyInstallPath, String password, BoostLoggerI logger) throws IOException {
+    public static String encrypt(String libertyInstallPath, String password, String encryptionKey, BoostLoggerI logger) throws IOException {
+        //Won't encode the password if it contains the aes flag
+        if (!password.contains("{aes}")) {
```

Comment:
What if the user provided an encrypted password that used something other than AES? Do we really want to encrypt it again using AES?

Reply:
Added a check for the 3 encryption types we support.